branch_project

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -xe
+
+. settings
+
+if [[ -z $1 ]] ; then
+	PROJECTS=$TAR_PROJECTS
+else
+	PROJECTS=$*
+fi
+
+PROJECTS=${PROJECTS/foreman-proxy/smart-proxy}
```

Comment:
Lines 5-11 are repeated fairly often across a few scripts, is there a way to capture them into a re-usable piece?

Reply:
I thought about the same, but struggled a bit. I'm now thinking about enhancing the ensure script to also output the project names and then use those as input. Will try something.

Reply:
How about this version?